config/users.php

```diff
@@ -40,6 +40,8 @@
             'ensureActive' => false,
             // default role name used in registration
             'defaultRole' => 'user',
+            // show verbose error to users
+            'ShowVerboseError' => false,
```

Comment:
Config should be in camelCase form, showVerboseError instead of ShowVerboseError